deploy/test/helm-props/openshift/helm-app.yaml

```diff
@@ -0,0 +1,26 @@
+# This app is created to run end-to-end with the Secrets Provider Job
```

Comment:
why don't we use the same yaml for OCP and k8s?

Reply:
Because the structure and apiversions are different. This breakdown is how we do it with our tests; a separate yaml for k8s and another one for openshift. Additionally the actions you take on them are different (scale, removal, etc) so wanted to continue this pattern

Reply:
Can you give an example?
I thought OCP can handle k8s yaml

Reply:
Sure
Role in k8s takes `rbac.authorization.k8s.io/v1`
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
```
Role in Openshift takes `v1`
```
apiVersion: v1
kind: Role
```

Also in k8s the deployment kind is `Deployment` and in Openshift it is `deploymentconfig`